### ISSUE TITLE
Other languages support

### ISSUE THREAD

**Issue by d07RiV** [Exslims/MercuryTrade#347]
Can't find an issue about it, but if you use official site to whisper other people, it will use the native language of their client. Mercury trade doesn't detect those as trade whispers if they're not English.

(It also skips some English whispers too, not sure if it's related)

**Comment by ghost** [comment]
Yes this has been ongoing for a while now.  Same issue when using other trade helper apps (POE-Trades-Companion, TraderForPoe, etc).

**Comment by MasterOfBytes** [comment]
I would really like to help here, as i usually want to play the game in german.

I should get the thing done, but i don't get the repo compiled into a single jar, to test my code.

Maybe someone just can help me to get my local repo startet, then i could do the code ( at least for German )
